priv/repo/migrations/20231016181014_user_account_update.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Game.Repo.Migrations.UserAccountUpdate do
+  use Ecto.Migration
+
+  def change do
+    alter table(:users) do
+      # add unique username
+      add :username, :string, null: false
```

Comment:
if you add `null: false` here -> what does happen to users that are already registered?
I would assume the migration fails and therefore it dies during deployment